### PR TITLE
(almost)automated CSS format

### PR DESCRIPTION
**Pull request by necessarily-equal**

### CODE REVIEW

File: ui/transition_loading.rml [hunk 1]
```diff
@@ -66,17 +67,18 @@
...
 
-			statusbox, percentage {
+			statusbox,
+			percentage {
 				color: white;
 				font-size: 1.5em;
 				font-weight: bold;
 				/* Dejavu has monospaced numbers, but Roboto does not
-				   This stops the the percentage number "jiggling" left and right */
+							   This stops the the percentage number "jiggling" left and right */
```
Comment (slipher) [inline ui/transition_loading.rml:76]: The thing messed up the formatting here

**Review by slipher — COMMENTED** [review]
Reformatting stuff is always controversial because of effectively erasing commit history. If we must do it, I'd rather after the 0.53 merge

File: ui/transition_loading.rml (hunk 1 above)
Comment (necessarily-equal) [reply ui/transition_loading.rml]: Fixed

**Comment by necessarily-equal** [comment]
I'd like merging this before 0.53 because it would mean mod have less files to distribute if they are based in master. And less conflicts if anyone starts working on unvanquished-0.53.0 instead of master.

**Comment by bmorel** [comment]
After thinking about it, I think changes to only change code-style are just a problem, and solve nothing. Please link this to me if I ever attempt to do so.
This makes everything more complicated for nothing.
Instead, we should define a script doing the job for us. This is not for humans to do.

**Comment by necessarily-equal** [comment]
I don't think this will ever have to be done twice.

For a start, CSS syntax has the benefit of being much simpler than C++, so it's much less ambiguous which style should be followed. Then, this is mostly about unifying it once to make it easier to read, to avoid wholly unreadable things and unify it once. I don't care too much if it drifts a bit with something slightly different but readable, like someone using `.5em` instead of `0.5em`.

And about using an automatic formatter, well while some css is in its own css files and is easily processable, some is in html files and is thus harder to process automatically

**Comment by slipher** [comment]
> I'd like merging this before 0.53 because it would mean mod have less files to distribute if they are based in master.

Huh?

> And less conflicts if anyone starts working on unvanquished-0.53.0 instead of master.

I've seen at least one RML change go into master recently, so merging the current PR will probably create conflicts. Merging after 0.53 (when there will be only a single existing branch) would avoid conflicts.

**Comment by DolceTriade** [comment]
fwiw, you can ignore commits when doing git blame for history. I don't think rewrite/reformat commits are as breaking as they used to be. However, ideally we do them all in a single/few commits so we can ignore these commits easily.

**Comment by DolceTriade** [comment]
+1 to merging after we release 0.53. Let's move this to 0.54.

**Comment by necessarily-equal** [comment]
Rebased. I hope nothing went wrong when rebasing.

**Comment by necessarily-equal** [comment]
> What the CSS code needs is […] a real cleanup

Well yes, but that would be easier to do if my eyes don't bleed. And the cleanup PR would be easier to read if they don't include formating changes too.

File: ui/hud_human.rml [hunk 2]
```diff
@@ -184,30 +187,27 @@
...
-			unlocked_items
-			{
+			unlocked_items {
 				width: 100%;
 				height: 50%;
 				/* Two-colour system. HACK
-					locked colour = CSS cell-color
-					unlocked colour = CSS color
-				 */
+								locked colour = CSS cell-color
```
Comment (slipher) [inline ui/hud_human.rml:201]: This spacing seems bad. Otherwise LGTM

**Review by DolceTriade — APPROVED** [review]
Thanks for fixing.

File: ui/hud_human.rml (hunk 2 above)
Comment (necessarily-equal) [reply ui/hud_human.rml:201]: fixed

**Pull request merged**